gittyup.nim

```diff
@@ -1349,8 +1144,8 @@ proc checkoutHead*(repo: GitRepository;
       options = cast[ptr git_checkout_options](sizeof(git_checkout_options).alloc)
     try:
       # setup our checkout options
-      withResultOf git_checkout_options_init(options,
-                                             GIT_CHECKOUT_OPTIONS_VERSION):
+      withResultOf git_checkout_options_init(
+        options, GIT_CHECKOUT_OPTIONS_VERSION):
```

Comment:
Okay, do we actually support these properly now?  I'm losing track of the status on these.

Reply:
What specifically do you mean by "these"?

Reply:
The constants.  I thought they were macros but now I'm not sure.

Reply:
There was no progress on the macro wrapping, so `GIT_CHECKOUT_OPTIONS_VERSION` is a constant. When hcparse properly supports macro wrapping, you could just delete the `const` section, and it will pick up definitions from the wrapper.

Reply:
Updated https://github.com/haxscramper/hcparse/issues/21 - I will track macro support progress in this issue (right now it is just a todo list, and I haven't started any real work).

Reply:
Works for me. :+1: